.circleci/config.yml

```diff
@@ -0,0 +1,121 @@
+version: 2.1
+orbs:
+  node: circleci/node@4.1.0
+
+references:
+  persist-workspace: &persist-workspace
+    persist_to_workspace:
+      root: ~/source
+      paths:
+        - .
+
+  attach-workspace: &attach-workspace
+    attach_workspace:
+      at: .
+
+defaults: &defaults
+  working_directory: ~/source
+  docker:
+    - image: circleci/node:latest
+  resource_class: medium+
+
+jobs:
+  lint-test:
+    <<: *defaults
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - v1-dependencies-{{ checksum "package.json" }}
+            # fallback to using the latest cache if no exact match is found
+            - v1-dependencies-
+
+      - run:
+          name: Install dependencies
+          command: yarn install
+      - run:
+          name: Run linting
+          command: yarn lint
+      - run:
+          name: Run tests
+          command: yarn test
+
+      - save_cache:
+          paths:
+            - node_modules
+          key: v1-dependencies-{{ checksum "package.json" }}
+      - *persist-workspace
+  build:
+    <<: *defaults
+    steps:
+      - *attach-workspace
+      - restore_cache:
+          keys:
+            - v1-dependencies-{{ checksum "package.json" }}
+            # fallback to using the latest cache if no exact match is found
+            - v1-dependencies-
+
+      - run: yarn install
+      - run:
+          name: Run build
+          command: yarn build
+          no_output_timeout: 20m
+
+      - *persist-workspace
+  release-publish:
+    <<: *defaults
+    steps:
+      - add_ssh_keys
+      - *attach-workspace
+      - restore_cache:
+          keys:
+            - v1-dependencies-{{ checksum "package.json" }}
+            # fallback to using the latest cache if no exact match is found
+            - v1-dependencies-
+      
+      - run:
+          name: Setup .env
+          command: echo -e "GH_TOKEN=${CIRCLECI_MOONSTONE_RELEASE_TOKEN}\nNPM_TOKEN=${NPMJS_TOKEN}" > ~/source/.env
+      - run:
+          name: Authenticate with registry
+          command: echo "//registry.npmjs.org/:_authToken=$NPMJS_TOKEN" > ~/source/.npmrc
+      - run: yarn install
+      - run:
+          name: Run build
+          command: yarn build
+          no_output_timeout: 20m
+      - run:
+          name: Release and publish
+          command: |
+            git config user.email "jahia-ci@jahia.com"
+            git config user.name "Jahia CI"
+            git fetch --tags
+            yarn auto-changelog
+            yarn auto-version
+            npm publish --access public
+            git push --follow-tags --set-upstream origin master
```

Comment:
Out of curiosity, this does not trigger another circleci run since you're pushing something back to master ?

Reply:
The commit that's generated by auto release has the [skip ci] in the commit messages.